# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/ScaledFontBase.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/2d/ScaledFontBase.cpp
--- a/gfx/2d/ScaledFontBase.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/2d/ScaledFontBase.cpp	Wed May 22 22:27:37 2019 +0000
@@ -6,7 +6,7 @@
 
 #include "ScaledFontBase.h"
 
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 
 #ifdef USE_SKIA
 #  include "PathSkia.h"
@@ -36,7 +36,7 @@
 ScaledFont::~ScaledFont() { sDeletionCounter++; }
 
 AntialiasMode ScaledFont::GetDefaultAAMode() {
-  if (gfxPrefs::DisableAllTextAA()) {
+  if (StaticPrefs::DisableAllTextAA()) {
     return AntialiasMode::NONE;
   }
 